packages/polyfills/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "@waku/polyfills",
```

Comment:
@waku-org/js-waku-developers how do you like the name? It is intended for `ReactNative` polyfilling. 
I'd like to avoid long two worded names if possible.

Reply:
ah, considering it's just for react-native, i would personally prefer to have that part of the package name for max verbosity

maybe in the future when we have different kinds of polyfills, this name would make more sense :D

Reply:
renamed to `react-native-polyfills`